docs/src/app/components/pages/customization/themes.md

```diff
@@ -0,0 +1,229 @@
+We changed how themes work in v0.12.0 (check out
+[release log](https://github.com/callemall/material-ui/releases/tag/v0.12.0) for more details).
+There are now two kinds of themes in Material-UI: **base theme** and **mui theme**.
```

Comment:
I agree, let's work on this in a new PR. I may even suggest that maybe we can remove the note about the theme changing in `0.12.0` because the link to the change log may confuse people more.